application/microservices/entando-hub-catalog-ms/src/test/java/com/entando/hub/catalog/entity/PrivateCatalogApiKeyEntityTest.java

```diff
@@ -0,0 +1,25 @@
+package com.entando.hub.catalog.entity;
+
+import com.entando.hub.catalog.persistence.entity.PrivateCatalogApiKey;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static com.entando.hub.catalog.service.PrivateCatalogApiKeyGeneratorHelper.generatePrivateCatalogApiKeyEntity;
+
+@SpringBootTest
```

Comment:
Is it required by the test? It seems to me that it is not involving any spring component

Reply:
Annotation removed